tkupchyn/homework_04/hashing_algorithm.py

```diff
@@ -0,0 +1,14 @@
+def count_checksum(word):
+    return sum(ord(letter) * (index + 1) for index, letter in enumerate(word))
+
+
+def group_words_by_checksum(words_to_order, number_of_groups):
+    result = {}
+
+    for word in words_to_order:
+        group = count_checksum(word) % number_of_groups
+        if group not in result:
+            result[group] = []
+        result[group].append(word)
```

Comment:
you can try 

result.setdefault(group, []).append(word)    # it returns value if key exists, or creates key with specified value and returns it


also you can try:

from collections import defaultdict
result =  defaultdict(list)
result[group].append(word)